.github/workflows/container_build.sh

```diff
@@ -23,6 +28,7 @@ ${HOME}/miniconda3/bin/conda activate hexrd
 
 # Install conda build and create output directory
 ${HOME}/miniconda3/bin/conda install --override-channels -c conda-forge conda-build -y
+${HOME}/miniconda3/bin/conda install -c conda-forge pybind11 -y
```

Comment:
Let's try to add these to the [build dependencies](https://github.com/HEXRD/hexrd/blob/f7e2abda823596d37e5f9bf139453c6294bc5310/conda.recipe/meta.yaml#L16-L21) instead of having an extra install/compile step beforehand.